app/src/main/assets/html/error/unable-to-connect.html

```diff
@@ -0,0 +1,27 @@
+<!doctype html>
+<html lang="en">
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="css/styles.css">
+    <title>Unable to Connect</title>
+    <body>
+        <script src="js/error.js"></script>
+        <div class="container">
+            <img class="icon" src="images/icon-errorpage-unabletoconnect.svg" alt="Unable to Connect">
+
+            <h1>Unable to Connect</h1>
+
+            <p>Firefox Reality can’t establish a connection to the
+            <br/>server at <b id="errorURL">www.abc.com</b></p>
```

Comment:
can you remove the `<br/>`? and add a period at the end: `</b></p>` -> `</b>.</p>`?